infra-generation-pipeline.yml

```diff
@@ -0,0 +1,143 @@
+trigger:
+  branches:
+    include:
+    - master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  group: 'spk-infra-hld-vg'
```

Comment:
Needs a `-` for listing the group in the yaml. Fails in the CLI scripts

Reply:
Good eye!